modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -52,6 +52,39 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
 
 
 
+== Bootstrap End-of-Life {centos} 6 Clients
+
+{centos} 6 is now at end-of-life, and the images provided in the client repository for this operating system are out of date.
+Bootstrapping new {centos} 6 clients using these packages will fail.
+This failure will not happen on {centos} 6 clients that are already installed and bootstrapped.
+
+If you need to bootstrap new {centos} 6 clients you can edit the existing repositories to reflect the correct URL.
+
+
+
+.Procedure: Troubleshooting Bootstrapping a New {centos} 6 Client
+. On the {centos} 6 client, at the command prompt, open the ``CentOS-Base.repo`` file, located in the `/etc/yum/repos.d/` directory.
```

Comment:
In fact, shouldn't the user do this change for more files than `CentOS-Base.repo`?

Please test the procedure yourself. No need for a SUSE Manager Server. Create a CentOS6 VM or start a docker container and give it a try :-)

Reply:
I did. The original text says to remove `CentOS-Base.repo`, `CentOS-Debuginfo.repo`, and `CentOS-fasttrack.repo`. When I looked at those files, only the base one has entries for the baseurl. So when we changed the procedure to just editing the file rather than deleting/re-writing, it seemed sensible not to worry about the other two. I guess we could still delete them, though, if you think that's important?

Reply:
> I guess we could still delete them, though, if you think that's important?

What I mean is that, as @mcalmer suggested somewhere else on this PR, we should give then `CentOS-Base.repo` as an example, and request the user to do the changes for the others files as welll, or disable them.

Otherwise I think bootstrapping from client will work (works even if some repositories fail to refresh, as long as packages are available), but maybe not from the WebUI (not 100% sure)

BTW, for both base and fastrack I don't see `baseurl` as you mention below, but `mirrorlist` (`baseurl` is commented)

```
[root@824fbdab45b8 yum.repos.d]# grep -ir 'mirrorlist' *
CentOS-Base.repo:# If the mirrorlist= does not work for you, as a fall back you can try the 
CentOS-Base.repo:mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=os&infra=$infra
CentOS-Base.repo:mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=updates&infra=$infra
CentOS-Base.repo:mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=extras&infra=$infra
CentOS-Base.repo:mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=centosplus&infra=$infra
CentOS-Base.repo:mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=contrib&infra=$infra
CentOS-fasttrack.repo:mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=fasttrack&infra=$infra
```
Are you sure in your tests you had `baseurl=`? I am 99.99% sure it should be `mirrorlist=` (but the hcange you suggest to `baseurl=` seems to be fine.

I'd suggest you try again to apply the instructions, and at the end run a `yum search vim` to see what if you can get the results without errors.

Reply:
I have run the procedure as written, with success. I only changed the baseurl entry in the base repo.

Reply:
Are you 100% sure the original line was `baseurl=` and not `mirrorlist=` as I see in my docker container?

Reply:
Yes, I only changed the baseurl entry.